### PR TITLE
MQTT server changes

### PR DESCRIPTION
**Pull request by hawk4031**
**Describe at a high level the solution you're providing**
All code is modified to make use of a VerneMQ install for the MQTT broker instead of Watson IoT

**Is this a patch, a minor version change, or a major version change**

**Is this related to an open issue?**

**Additional context**
The Watson IoT service will be going away eventually, so the modifications in this and other repositories reflect a new MQTT broker (VerneMQ) that is deployed alongside the rest of the solution

### CODE REVIEW

**Comment by hawk4031** [comment]
@upkarlidder any questions or comments?

File: action/Dockerfile.dev [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM docker.io/node:12-alpine
+#FROM docker.io/node:12-alpine
```
Comment (upkarlidder) [inline action/Dockerfile.dev:1]: Should we pin this to a SHA? I remember we got bit in RulesDecision as the version got updated when we built and deployed.

File: action/index.js [hunk 2]
```diff
@@ -4,26 +4,22 @@ var mqtt = require("mqtt");
...
     const client = mqtt.connect(options);
 
-    client.on("connect", function (err) {
-      if (err) {
-        console.log(`error connecting ${params.IOT_DEVICE_ID}`);
-        console.log(JSON.stringify(err));
-      }
+    client.on("connect", function (packet) {
+      // if (err) {
+      //   console.log(`error connecting ${params.IOT_DEVICE_ID}`);
```
Comment (upkarlidder) [inline action/index.js:20]: I wonder if we should add conditional formatting with something like [pino](https://www.npmjs.com/package/pino) in combination with `process.env.NODE_ENV`? Will be good to capture and monitor logs in production?

**Review by upkarlidder — APPROVED** [review]
Added some small comments, but looks good to me!

**Pull request merged**